jdk11-gradle4/Dockerfile

```diff
@@ -3,15 +3,15 @@ FROM openjdk:11-jdk
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 4.10.3
+ENV GRADLE_VERSION 5.6.1
```

Comment:
Since this file path contains `jdk11-gradle4`, I think that it shouldn't be changed...